lib/kitchen/directions/bake_eoc_section_content/remove_section_title.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Kitchen
+  module Directions
+    module RemoveSectionTitle
+      def self.v1(section:)
+        section.first('h3').trash
```

Comment:
It has data-type="title" in all the books I've looked at, but the spec for Key Equations doesn't include that data type. Not sure if that means there's a book that's an exception? I agree that this seems like a better selector, but there's a chance it could be a breaking change.

Reply:
Ok I yield to you on this

Reply:
haha I did change it to data-type="title" ... 🤞